### DISCUSSION TITLE
Percentile Pooling for Global Output

### DISCUSSION THREAD

**Discussion by khainguyen20** [pyg-team/pytorch_geometric#8838] [General]
I have a quick question. For the global pooling operations, I believe there are only 3 for now, which is global max, global average, and global add. Is there anything where instead, we can get the percentiles of the node features (25%, 50%, etc.)? I believe this way, we lose less information and is also less prone to outliers. I'm trying to implement this but I'm having a bit difficulty with getting the percentiles for the specific graphs in a batch. Would there be a way to implement this type of global pooling operation? Please let me know. Any help would be appreciated!

**Comment by rusty1s** [comment]
There exists a `QuantileAggregation` operator you can use, see [here](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.aggr.QuantileAggregation.html#torch_geometric.nn.aggr.QuantileAggregation).

**Comment by khainguyen20** [comment]
This is what I was looking for. Thank you!
